### PR TITLE
New Package: Smartedu(国家中小学智慧教育平台)-1.0.0

### PR DESCRIPTION
**Pull request by CoolPlayLin**
- [x] Have you signed the [Contributor License Agreement](https://cla.opensource.microsoft.com/microsoft/winget-pkgs)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/microsoft/winget-pkgs/pulls) for the same manifest update/change?
- [x] This PR only modifies one (1) manifest
- [x] Have you [validated](https://github.com/microsoft/winget-pkgs/blob/master/AUTHORING_MANIFESTS.md#validation) your manifest locally with `winget validate --manifest <path>`?
- [x] Have you tested your manifest locally with `winget install --manifest <path>`?
- [x] Does your manifest conform to the [1.4 schema](https://github.com/microsoft/winget-pkgs/tree/master/doc/manifest/schema/1.4.0)?

Note: `<path>` is the name of the directory containing the manifest you're submitting.

-----

 ###### Microsoft Reviewers: codeflow:open?pullrequest=https://github.com/microsoft/winget-pkgs/pull/110342&drop=dogfoodAlpha

### CODE REVIEW

**Comment by stephengillie** [comment]
Automatic Validation ended with:
image

File: manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Smartedu.Smartedu
+PackageVersion: 1.0.0
+Installers:
+- Architecture: x86
```
Comment (stephengillie) [inline manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.installer.yaml:7]:
```suggestion
- Architecture: x64
```

**Comment by CoolPlayLin** [comment]
The download page: [click](https://basic.smartedu.cn/download_app)

**Comment by CoolPlayLin** [comment]
image
 The URL is correct

**Comment by CoolPlayLin** [comment]
@mdanish-kh Could you review this for me?

**Comment by CoolPlayLin** [comment]
@stephengillie

**Comment by CoolPlayLin** [comment]
I think it only shows in China

**Comment by CoolPlayLin** [comment]
> I think it's only available in China

Refers to this button

**Comment by CoolPlayLin** [comment]
@stephengillie

**Comment by CoolPlayLin** [comment]
@stephengillie @ImJoakim @russellbanks @mdanish-kh

**Comment by stephengillie** [comment]
> I think it's only available in China

Unfortunately, to be a good fit into the WinGet public manifests, packages have to be available for download from almost any terrestrial location, at any time, by almost any number of anonymous users from the same address.

**Comment by CoolPlayLin** [comment]
Only the web, and the download URL is available all over the world@stephengillie

**Comment by SpecterShell** [comment]
Although the download link is fetched, it would be hidden for oversea users.
https://grey-config.ykt.cbern.com.cn/web_config.json
image

**Comment by CoolPlayLin** [comment]
> Although the download link is fetched, it would be hidden for oversea users. https://grey-config.ykt.cbern.com.cn/web_config.json image

CC. @stephengillie

**Comment by CoolPlayLin** [comment]
> Generally, to be a good fit into the WinGet public manifests, packages have to be available for download from almost any terrestrial location, at any time, by almost any number of anonymous users from the same address.
@stephengillie 
As you see,The test has been passed, so the download URL is available all over the world

**Comment by SpecterShell** [comment]
> As you see,The test has been passed, so the download URL is available all over the world

It is hard to say if the download button is hidden (or not shown) by intention although. 🤔

**Comment by CoolPlayLin** [comment]
image

Because it is beta version, I think that's why it only shows download button in China

**Comment by CoolPlayLin** [comment]
@Trenly

File: manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Smartedu.Smartedu
+PackageVersion: 1.0.0
+PackageLocale: en-US
+Publisher: Ministry of Education of the People's Republic of China
+PublisherUrl: https://basic.smartedu.cn/
+PublisherSupportUrl: https://basic.smartedu.cn/
+PrivacyUrl: https://basic.smartedu.cn/
```
Comment (Trenly) [inline manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.en-US.yaml:10]:
```suggestion
PublisherUrl: https://www.smartedu.cn/
PublisherSupportUrl: https://www.smartedu.cn/special/helpcenter
# PrivacyUrl:
```

These three URLS should not be the same as the PackageURL. They should link to the website of the publisher, the support page or contact us form, and the privacy policy directly

File: manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.zh-CN.yaml [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+PackageIdentifier: Smartedu.Smartedu
+PackageVersion: 1.0.0
+PackageLocale: zh-CN
+Publisher: 中华人民共和国教育部
+PublisherUrl: https://basic.smartedu.cn/
+PublisherSupportUrl: https://basic.smartedu.cn/
+PrivacyUrl: https://basic.smartedu.cn/
+Author: 教育部教育技术与资源发展中心
+PackageName: 国家中小学智慧教育平台
```
Comment (Trenly) [inline manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.zh-CN.yaml:12]:
```suggestion
PackageName: 智慧中小学
```
This should match what is written to control panel

File: manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.zh-CN.yaml [hunk 4]
```diff
@@ -0,0 +1,19 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: Smartedu.Smartedu
+PackageVersion: 1.0.0
+PackageLocale: zh-CN
+Publisher: 中华人民共和国教育部
```
Comment (Trenly) [inline manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.zh-CN.yaml:7]:
```suggestion
Publisher: 智慧中小学
```
This should match what is written into control panel

File: manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.installer.yaml [hunk 5]
```diff
@@ -0,0 +1,12 @@
+# Created using wingetcreate 1.2.6.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.4.0.schema.json
+
+PackageIdentifier: Smartedu.Smartedu
+PackageVersion: 1.0.0
+Installers:
+- Architecture: x64
+  InstallerType: nullsoft
+  InstallerUrl: https://download.ykt.cbern.com.cn/pc_ver/%E6%99%BA%E6%85%A7%E4%B8%AD%E5%B0%8F%E5%AD%A6%20Setup%201.0.0.exe
+  InstallerSha256: EDC09E439189E11D95C7BFD6265B7A5DF5AE6D2E152B6D5970F65B49254C54AF
```
Comment (Trenly) [inline manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.installer.yaml:10]:
```suggestion
  InstallerSha256: EDC09E439189E11D95C7BFD6265B7A5DF5AE6D2E152B6D5970F65B49254C54AF
  ProductCode: 'c6cc9519-7239-555e-b960-aa99cc131aac'
```
The product code will help identifying when the package is installed

File: manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.en-US.yaml [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+Author: Center for Educational Technology and Resource Development of MOE
+PackageName: Smartedu
+PackageUrl: https://basic.smartedu.cn/
+License: Free Software
```
Comment (Trenly) [inline manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.en-US.yaml:14]:
```suggestion
License: Freeware
```

File: manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.zh-CN.yaml [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+PublisherUrl: https://basic.smartedu.cn/
+PublisherSupportUrl: https://basic.smartedu.cn/
+PrivacyUrl: https://basic.smartedu.cn/
```
Comment (Trenly) [inline manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.zh-CN.yaml:10]:
```suggestion
PublisherUrl: https://www.smartedu.cn/
PublisherSupportUrl: https://www.smartedu.cn/special/helpcenter
# PrivacyUrl:
```

These three URLS should not be the same as the PackageURL. They should link to the website of the publisher, the support page or contact us form, and the privacy policy directly

File: manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.zh-CN.yaml [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+Author: 教育部教育技术与资源发展中心
+PackageName: 国家中小学智慧教育平台
+PackageUrl: https://basic.smartedu.cn/
+License: 免费软件
+Copyright: Copyright © 2022 主办：中华人民共和国教育部  网站运维：教育部教育技术与资源发展中心（中央电化教育馆）
+CopyrightUrl: https://basic.smartedu.cn/
```
Comment (Trenly) [inline manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.zh-CN.yaml:16]: This should be a direct link to the package copyright

File: manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.en-US.yaml [hunk 9]
```diff
@@ -0,0 +1,19 @@
...
+Copyright: Copyright © 2022 Host：Ministry of Education of the People's Republic of China Website：Center for Educational Technology and Resource Development of MOE
+CopyrightUrl: https://basic.smartedu.cn/
```
Comment (Trenly) [inline manifests/s/Smartedu/Smartedu/1.0.0/Smartedu.Smartedu.locale.en-US.yaml:16]: This should be a direct link to the package copyright

**Review by Trenly — CHANGES_REQUESTED** [review]

**Comment by CoolPlayLin** [comment]
What?

**Comment by CoolPlayLin** [comment]
@Trenly I also have another pr needs to review:

https://github.com/microsoft/winget-pkgs/pulls/CoolPlayLin

**Comment by CoolPlayLin** [comment]
@Trenly What?

**Comment by Trenly** [comment]
> @Trenly What?

It seems that the website didn't respond to the automated validation request. Stephen will have to look into it more

**Comment by CoolPlayLin** [comment]
？

**Comment by stephengillie** [comment]
> Only the web, but the download URL is available all over the world @stephengillie

Ah. This prevents me from navigating from the PackageUrl to the InstallerUrl. 
- Thinking on this, any user at almost any terrestrial location should be able to navigate from the PackageUrl to the InstallerUrl.

**Comment by CoolPlayLin** [comment]
This software is only used in China

**Comment by stephengillie** [comment]
> This software is only used in China

The limited availability of this package unfortunately makes it a poor choice for inclusion into the WinGet public manifests.

**Pull request closed without merging**